controllers/users.ts

```diff
@@ -0,0 +1,82 @@
+import { NextFunction, Request, Response } from 'express';
+import httpStatus from 'http-status';
+import { isValidObjectId } from 'mongoose';
+import { userModel } from '../models/users';
+
+export const createUser = async (request: Request, response: Response, next: NextFunction) => {
+	const data = request.body;
+
+	try {
+		const newUser = await userModel.create(data);
```

Comment:
Hash the password bruv (this also means hashing in login)

Reply:
Removed the user creation route. I will add it back as part of the user registration route which will hash the password etc